database/index.js

```diff
@@ -53,7 +53,7 @@ var updateReview = async (id, dataToChange) => {
   return review;
 };
 
-var createReview = async (reviewData) => {
+var createReview = async (reviewData, callback) => {
```

Comment:
It may benefit you to move "awards" into its own table

Reply:
Will keep this in mind when I implement my own database rather than the legacy database!